index.js

```diff
@@ -4,14 +4,15 @@ const fp = require('fastify-plugin');
 const { default: Client } = require('@immobiliarelabs/dats');
 const doc = require('@dnlup/doc');
 const { hrtime2ns, hrtime2ms, hrtime2s } = require('@dnlup/hrtime-utils');
+const STATSD_METHODS = ['counter', 'timing', 'gauge', 'set'];
```

Comment:
we would remove `on`?

Reply:
Yes, it is not a method of Dats, I don't know why it was there.